PLUGINS.md

```diff
@@ -0,0 +1,29 @@
+# Plugins
+
+## How to implement plugin
+
+Plugin is a class which inherits from `ariadne_codegen.plugins.base.Plugin`. Plugin's instance is created once, at the beginning of `ariadne-codegen` command, with following arguments:
+- `schema: GraphQLSchema`: parsed graphql schema
+- `config_dict: Dict`: parsed `pyproject.toml` file represented as a dictionary
+
+To handle specific hooks class needs to override [base methods](#hooks). Not changed methods have no effect on generated code and don't raise `NotImplementedError`.
+
+
+## Plugins lookup
+
+Plugin can be provided to `ariadne-codegen` in 2 ways:
+
+1. Providing full import path to class, eg. `my_package.my_plugins.MyPlugin`.
+
+2. Providing only package name, eg. `my_package`. In this case all plugins from selected package will be used. For class to be accessible this way, it needs to be in package's public api. Plugin has to be importable from package, eg. `from my_package import MyPlugin` has to work.
+
+
+## Hooks
+
+### generate_init_module
```

Comment:
Lets add an example plugin that uses `generate_init_module` to add something to output file. For example, `__version__ = "0.1"`

Reply:
I added `## Example` section with implementation